TaskManager/Modules/Home/View/HomeViewController.swift

```diff
@@ -67,30 +71,42 @@ class HomeViewController: UIViewController, HomeView {
         tableView.register(UINib(nibName: String(describing: TaskTableViewCell.self), bundle: nil), forCellReuseIdentifier: TaskTableViewCell.identifier)
         tableView.delegate = self
         
-        dataSource = HomeDataSource(tableView: tableView, cellProvider: { (tableView, indexPath, itemIdentifier) -> UITableViewCell? in
+        dataSource = HomeDataSource(dataStorage: dataStorage, tableView: tableView, cellProvider: { (tableView, indexPath, itemIdentifier) -> UITableViewCell? in
             guard let cell = tableView.dequeueReusableCell(withIdentifier: TaskTableViewCell.identifier, for: indexPath) as? TaskTableViewCell else {return UITableViewCell()}
-            
-            cell.configureCell(viewModel: itemIdentifier)
+            switch indexPath.section {
+            case 0:
+                cell.configureCell(viewModel: itemIdentifier, backgroundColor: .systemBackground)
+                cell.circleLabel.backgroundColor = .systemBackground
+            case 1:
+                cell.configureCell(viewModel: itemIdentifier, backgroundColor: .taskManagerColor)
+                cell.changeCompletedTask()
+            default:
+                break
+            }
             return cell
         })
         snapshot = dataSource.snapshot()
     }
     
-    func showTask(tasks: [ModelTask]) {
-        let sections = [Section.Active, Section.Completed]
-        snapshot.appendSections(sections)
-
-        for section in sections {
-            let items = tasks.filter { task in
-                task.type == section
-            }
-            snapshot.appendItems(items, toSection: section)
+    private func configFetchController() {
+        fetchedResultsController = dataStorage.getFetchedResultsController()
+        fetchedResultsController.delegate = self
+        do {
+            try fetchedResultsController.performFetch()
+        } catch {
+            print("Fetch failed")
         }
-        dataSource.apply(snapshot, animatingDifferences: false)
     }
     
-    func appendItems(task: ModelTask, section: Section) {
-        snapshot.appendItems([task], toSection: section)
+    func showTask(tasks: [ModelTask]) {
+        snapshot.appendSections([Section.Active, Section.Completed])
+        for task in tasks {
+            if task.type == true {
+                snapshot.appendItems([task], toSection: Section.Active)
+            } else {
+                snapshot.appendItems([task], toSection: Section.Completed)
+            }
```

Comment:
```suggestion
        snapshot.appendItems([task], toSection: task.type ? .Active : .Completed)
```
You can make use of ternary operator